packages/nextra-theme-blog/src/mdx-theme.tsx

```diff
@@ -31,70 +32,28 @@ const H1 = ({ children }: { children?: React.ReactNode }) => {
     <>{showHeading ? ReactDOM.createPortal(children, ref.current!) : null}</>
   )
 }
+const createHeaderLink =
+  (Tag: `h${2 | 3 | 4 | 5 | 6}`) =>
+  ({ children, ...props }: ComponentProps<'h2'>): ReactElement => {
+    return (
+      <Tag {...props}>
+        <a
+          href={`#${props.id}`}
```

Comment:
I removed `github-slugger` since we don't need it as anchors are already well sluglified by `github-slugger` in `nextra`